charts/posthog/templates/_clickhouse.tpl

```diff
@@ -95,3 +95,17 @@ Return the ClickHouse image
 {{- define "posthog.clickhouse.image" -}}
 "{{ .Values.clickhouse.image.repository }}:{{ .Values.clickhouse.image.tag }}"
 {{- end -}}
+
```

Comment:
Why do we need those helpers?

Reply:
allows to use less amount of letters and this chart is using helper for other docker image.